code/_onclick/click.dm

```diff
@@ -96,7 +96,7 @@
 
 		return
 
-	if(!isturf(loc)) // This is going to stop you from telekinesing from inside a closet, but I don't shed many tears for that
+	if(!isturf(loc) && !find_holder_of_type(A, src)) // Can't touch anything from inside a locker/sleeper etc, unless it's inside our inventory.
```

Comment:
Makes it possible to attack something being held by a mob of your same type
But how are you going to attack that thing if the other mob is holding it